docs/specs/layers.md

```diff
@@ -71,3 +71,76 @@ decrease it ("stretching" the layer timeline).
     </script>
 </lottie-playground>
 
+The `tm` property specifies a time remap function, which offers full control over the precomp
```

Comment:
I'd create a new subsection for time remapping

Reply:
Done.  Also added a subsection for time stretch.